go-controller/go.mod

```diff
@@ -50,15 +50,15 @@ require (
 	gopkg.in/fsnotify/fsnotify.v1 v1.4.7
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
-	k8s.io/api v0.29.2
-	k8s.io/apimachinery v0.29.2
-	k8s.io/client-go v0.29.2
+	k8s.io/api v0.30.0
+	k8s.io/apimachinery v0.30.0
+	k8s.io/client-go v0.30.0
```

Comment:
Do you need to bump `k8s.io/component-helpers v0.29.2` below as well?

Reply:
turns out we don't need it but I bumped it explicitly to align